### ISSUE TITLE
DFG: ENA is mentioned many times even if it is not selected

### ISSUE THREAD

**Issue by StellaEggels** [nfdi4plants/dataplan#51]
Under 3.1 ENA and PRIDE (capitalized wrongly) are mentioned even if they are not selected. Under 3.2 ENA is also mentioned without being selected and the beginning of that sentence is not capitalized. Under 5.3 ENA again is mentioned in the prewritten text.

**Comment by xiaoranzhou** [comment]
For the last one, I can put it into the plancehoders, but for the first two, I think they are just giving examples related to sensitive data (section 3.2) and international discipline (section 3.1). I don't think we need to delete them.

**Comment by StellaEggels** [comment]
I don't know, let's say it's a phenotyping project and ENA is not at all relevant for that project, then I think it is strange to mention, even as an example

**Comment by xiaoranzhou** [comment]
Yes, maybe show ENA in phenotyping is not relevant, but do we show all the archive names when the project has multi-omics? Or we can remove all the e.g. here because it has already mentioned the "specialized technologies (as detailed above)"

**Comment by StellaEggels** [comment]
I would say the e.g. can be removed because of the "as detailed above" as you say
